src/thunderbots/software/ai/util.h

```diff
@@ -41,3 +43,16 @@
     {                                                                                    \
         return static_cast<CLASS>(~static_cast<UNDERLYING>(lhs));                        \
     }
+
+inline std::unordered_set<unsigned int> commaSeparatedListToSet(
+    const std::string& list_str)
+{
+    std::stringstream ss(list_str);
```

Comment:
Can we be more robust to strings not in csv format? Just logging and moving on is fine, but we don't want a stray character taking down AI (did this during testing)